src/custom.jl

```diff
@@ -16,12 +16,45 @@ function splitup(data::Vector{UInt8}, offsets, T::Type; skipbytes=0)
     out
 end
 
-# Custom, hardcoded streamers
-
+# Custom struct interpretation
 abstract type CustomROOTStruct end
 
-# KM3NeT
+reinterpret(vt::Type{Vector{T}}, data::AbstractVector{UInt8}) where T <: CustomROOTStruct = reinterpret(T, data)
+
+function interped_data(rawdata, rawoffsets, ::Type{J}, ::Type{T}) where {J <: JaggType, T <: CustomROOTStruct}
```

Comment:
Why is this forwarding needed for `T <: CustomROOTStruct`, I guess if there is no dispatch on the type like for `TLorenzVector` the basic `interped_data` from `root.jl` is called anyway, isn't it?